src/08-testing.md

```diff
@@ -43,41 +43,41 @@ test-suite hs-blog-gen-test
     hspec-discover:hspec-discover
 ```
 
-- `hs-source-dirs: test` - The directory of the source files for the test suite.
-- `main-is: Spec.hs` - The entry point to the test suite.
+- `hs-source-dirs: test` - The directory of the source files for the test suite
+- `main-is: Spec.hs` - The entry point to the test suite
 - `other-modules` - The modules in our test suite.
-  Currently commented out because we haven't added any yet.
+  Currently commented out because we haven't added any yet
 - `build-depends` - The packages we are going to use:
   - [`base`](https://hackage.haskell.org/package/base) -
-    The standard library for Haskell, as we've used before.
-  - [`hspec`](https://hackage.haskell.org/package/hspec) -
-    The test framework we are going to use.
+    The standard library for Haskell, as we've used before
+  - [`Hspec`](https://hackage.haskell.org/package/hspec) -
```

Comment:
I thought the Package it called 'Hpack' but the package author himself is also not consequent here : https://github.com/sol/hpack#hpack-a-modern-format-for-haskell-packages

Reply:
The library is called Hspec, but the package is named hspec with lowercase h. Since packages is Haskell are case-sensitive ([you won't fine `Hspec` on hackage](https://hackage.haskell.org/package/Hspec)) I think it's fine to refer to it as Hspec when we casually talk about it, but use `hspec` when we refer to the name of the package.

Reply:
ok